package.json

```diff
@@ -51,7 +51,6 @@
     "jshint-stylish": "^1.0.0",
     "merge-stream": "^0.1.7",
     "mocha": "^2.1.0",
-    "napa": "^1.2.0",
```

Comment:
Could you also remove `"swfobject-npm": "^2.3.0-1"` below? we used it for flash detection.